pkg/compiler/gen.go

```diff
@@ -441,11 +441,28 @@ func (comp *compiler) genFieldArray(fields []*ast.Field, argSizes []uint64) []pr
 	return res
 }
 
+func (comp *compiler) genFieldDir(f *ast.Field) (bool, prog.Dir) {
```

Comment:
In Go such "ok"/"has" bool is commonly returned as the second return value, e.g.:
```
v, ok := <-channel
v, ok := map[key]
x, ok := iface.(Type)
```
Use the same convention.

Reply:
[Done](https://github.com/google/syzkaller/pull/2008/commits/82ffbda67776502531b39b560e33ac8956d90827#diff-5a1b8af690df33a2e42dabbf3d5db46dL444-R456)